api/app.js

```diff
@@ -1,8 +1,12 @@
 const express = require('express')
-const { getTopics } = require('./controllers/api.topics.contoller')
+const { getTopics, getArticleById } = require('./controllers/api.topics.contoller')
+const { handleServerError } = require('./error_handlers/errors')
```

Comment:
Usually, in JavaScript, we use camelCase, not snake_case - just something to keep in mind as best practice 😃